category/index.html

```diff
@@ -9,12 +9,13 @@
 	<div class="col-12 col-lg-9 mx-auto">
 		<div class="row">
 			<ul class="list-group w-100 text-center">
```

Comment:
Without `li` or `ol`, this shouldn't be `ul`

Reply:
changed to div.